src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -143,6 +143,72 @@ export default [
         },
         getDescription: () => {},
     },
+    {
+        isAnonymousAction: false,
+        textTranslateKey: 'reportActionContextMenu.subscribeToThread',
+        // textTranslateKey: lodashGet(reportAction, 'childReportNotificationPreference', '0'),
+        icon: Expensicons.Bell,
+        successTextTranslateKey: '',
+        successIcon: null,
+        shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID) => {
+            const childReportNotificationPreference = lodashGet(reportAction, 'childReportNotificationPreference', '');
+            const subscribed = childReportNotificationPreference && childReportNotificationPreference !== 'hidden';
+            if (type !== CONTEXT_MENU_TYPES.REPORT_ACTION) {
+                return false;
+            }
+            const isCommentAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && !ReportUtils.isThreadFirstChat(reportAction, reportID);
+            const isReportPreviewAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW;
+            const isIOUAction = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && !ReportActionsUtils.isSplitBillAction(reportAction);
+            return !subscribed && (isCommentAction || isReportPreviewAction || isIOUAction);
```

Comment:
i think we should not show this icon if the reportAction does not have a child since that does not make sense to me. Like whats the usecase to subscribe to a thread that does not exist?